docs/content/operator/common/monitoring.mdx

```diff
@@ -0,0 +1,102 @@
+After a successful setup of a node, the next important step is to set up monitoring for the node.
+Monitoring is important to keep track of the node's health and performance.
+
+## Network Monitoring
+All active validators for the testnet are listed on the [IOTA Explorer](https://explorer.rebased.iota.org/validators) website.
```

Comment:
that's kind of a random info here 🤷

Reply:
Removed in 0c0a6337ba4b90bca65d1713222fc05d4843e3bf